epictrack-web/src/components/project/ProjectForm/index.tsx

```diff
@@ -285,7 +288,7 @@ export default function ProjectForm({
               options={types || []}
               getOptionValue={(o: Type) => o?.id?.toString()}
               getOptionLabel={(o: Type) => o.name}
-              disabled={isSpecialFieldLocked}
+              disabled={isSpecialFieldLocked || !canEdit}
```

Comment:
Is it possible to put all the disable conditions together in one variable?

Reply:
done